<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>381</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QFrame" name="req_frame">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lb_request">
          <property name="styleSheet">
           <string notr="true">font-size: 12pt;</string>
          </property>
          <property name="text">
           <string>Запрос</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="btn_req_frame">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="btn_gps">
             <property name="text">
              <string>GPS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_imu">
             <property name="text">
              <string>IMU</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="mode_frame">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lb_mode">
          <property name="styleSheet">
           <string notr="true">font-size: 12pt;</string>
          </property>
          <property name="text">
           <string>Режим</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="btn_mode_frame">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="btn_mnl_mode">
             <property name="text">
              <string>Ручной</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_auto_mode">
             <property name="text">
              <string>Автономный</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_rmt_mode">
             <property name="text">
              <string>Дистанционный</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="pwr_mtr_frame">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lb_pwr_mnl">
          <property name="styleSheet">
           <string notr="true">font-size: 12pt;</string>
          </property>
          <property name="text">
           <string>Мощность</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_pwr_mnl">
          <property name="maximumSize">
           <size>
            <width>186</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="ctrl_mtr_frame">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lb_motor">
          <property name="styleSheet">
           <string notr="true">font-size: 12pt;</string>
          </property>
          <property name="text">
           <string>Двигатель</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ctrl_mtr">
          <property name="text">
           <string>Старт</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="conn_frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="conn_frame">
     <item>
      <widget class="QPushButton" name="btn_connect">
       <property name="text">
        <string>Подключиться</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_disconnect">
       <property name="text">
        <string>Отключиться</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="te_mnl_cmd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="select_params_com_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QComboBox" name="cb_com_dev">
       <property name="placeholderText">
        <string>не выбрано</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_baudrate">
       <property name="placeholderText">
        <string/>
       </property>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="text_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clean_textBrw">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
